app/src/main/java/cm/aptoide/pt/home/FakeBundleDataSource.java

```diff
@@ -46,11 +46,11 @@
 
     List<Application> tmp1 = new ArrayList<>();
     FeatureGraphicApplication aptoideFeatureGraphic =
-        new FeatureGraphicApplication("Aptoide", icon, 0, 1000, "cm.aptoide.pt", 300, icon);
+        new FeatureGraphicApplication("Aptoide", icon, 0, 1000, "cm.aptoide.pt", 300, icon, "", "");
```

Comment:
Repeatable values, maybe static final variables?

Reply:
In normal code, I would say yes. But I think in this fake bundle source, that is only used for testing, gets more flexibility and readability by not reusing some of its values. In this example there are also terrible names like tmp and tmp1 but they are strictly used for testing but I will rename them